src/core/HTTP/RestClient.ts

```diff
@@ -44,46 +47,45 @@ export default class RestClient {
 
     this.client = axios.create({
       baseURL: this.clientUrl,
-      url: workspaceUrl,
       headers: this.clientHeaders,
     })
   }
 
   public async getFiles<T>(options?: AxiosRequestConfig): Promise<FileResourceJSON[]> {
-    return this.handleResponse(await this.client.get('/files', options))
+    return this.handleResponse(await this.client.get(`${this.workspaceName}/files`, options))
   }
 
   public async getAllFiles<T>(): Promise<FileResourceJSON[]> {
-    let res = await this.client.get('/files')
+    let res = await this.client.get(`${this.workspaceName}/files`)
     let files: FileResourceJSON[] = this.handleResponse(res)
     while (res.data.next) {
-      // url is set to empty because next already has workspace
-      res = await this.client.get(res.data.next, { url: '' })
+      // url already has workspace
+      res = await this.client.get(res.data.nex)
```

Comment:
looks like this should be `.next`

Reply:
fixing one thing and breaking another, lol
ty